src/modules/launcher/Plugins/Microsoft.Plugin.WindowWalker/Components/ResultHelper.cs

```diff
@@ -97,6 +97,11 @@ private static string GetSubtitle(Window window)
                 subtitleText += $" - {Resources.wox_plugin_windowwalker_Desktop}: {window.Desktop.Name}";
             }
 
+            if (!window.Process.Responding)
+            {
+                subtitleText += $" [{Resources.wox_plugin_windowwalker_NotResponding}] ";
```

Comment:
```suggestion
                subtitleText += $" [{Resources.wox_plugin_windowwalker_NotResponding}]";
```

Sory, was my mistake.

And please switch place with lines 95 to 98 (Desktop subtitle part).

Reply:
Fixed in commit 36cb768